src/App.js

```diff
@@ -65,6 +62,20 @@ export default class App extends Component {
     setLoadingEntity: (loadingEntity) => {
       this.setState({ loadingEntity });
     },
+    setSetting: (settingKey, settingValue) => {
+      let states = {
+        settings: { ...this.state.settings, [settingKey]: settingValue },
+      };
+      try {
+        localStorage.setItem(settingKey, settingValue);
```

Comment:
use that package I mentioned and save the whole setting object (hopefully will be serialised)

Reply:
Yes, but it's not necessary anymore, is it? I already installed it, but now using an object will be more difficult

Reply:
You can smash the whole object in, it will serialize for you 
```
//set
localStorage("settings", settings)
//get
const settings = localStorage("settings")
```